plugin.xml

```diff
@@ -200,4 +200,22 @@
 
 
     </platform>
+    
+    <platform name="browser">
+
+        <config-file target="config.xml" parent="/*">
+            <feature name="Mixpanel">
+                <param name="browser-package" value="MixpanelPlugin"/>
+            </feature>
+        </config-file>
+
+        <js-module src="src/browser/MixpanelProxy.js" name="MixpanelProxy">
```

Comment:
thanks for the reference, will take a look at it soon.

meanwhile - the pr is not mergable because master needs to be merged into your branch.
can you do that?